### ISSUE TITLE
Feature request: Make Android Launcher the same feature reach and configurable as Desktop launchers.

### ISSUE THREAD

**Issue by krukhlis** [ja2-stracciatella/ja2-stracciatella#1341]
Hi guys,
If you compare Android Launcher and Desktop's -- you will find out that Android Launcher has only one configurable option -- location of Data folder. Whilst Desktop's launcher allows you to configure screen resolution, language, and tons of other parameters. 
Is it possible to provide similar functionality in Android Launcher?

Version tested: 0.17 release and 0.17.x nightlies.

**Comment by selaux** [comment]
Yes, this is definitely planned. Currently the Android version in early development still.

**Comment by ViaFerrata** [comment]
A related question:
is it even possible to use mods on Android, since they need to be activated via cmd or (desktop) launcher?

Or is there a workaround?

**Comment by lynxlynxlynx** [comment]
Likely, just add them to ja2.json manually.

**Comment by selaux** [comment]
Right now it is not easily possible, as the home directory where you would need to put the mods is not accessible easily on Android.

**Comment by ViaFerrata** [comment]
That's what I wanted to write as well.

Tried on Android 13, stracciatella is located in data/user/0/some_hash/io.github.ja2stracciatella.
Can't be found via USB on PC and actually I also couldn't find it with the usual Android file browsers.
Didn't try MiXplorer yet (5$) which seems to be able to access without root (?), otherwise you'd need to root the device first.

Thought about modyfing within the apk but the config file gets created upon the first app launch so it's not possible.
Extended guns with the mod would certainly be nice.
